instagram_private_api/endpoints/accounts.py

```diff
@@ -75,32 +75,38 @@ def current_user(self):
             ClientCompatPatch.user(res['user'], drop_incompat_keys=self.drop_incompat_keys)
         return res
 
-    def edit_profile(self, first_name, biography, external_url, email, phone_number, gender):
+    def edit_profile(self, username=None, first_name=None, biography=None,
```

Comment:
The username argument should be placed last. This preserves compatibility and also, I don't think many people will be changing their usernames often.

And have you tested actually changing the username?